flowapi/flowapi/qa_endpoints.py

```diff
@@ -0,0 +1,205 @@
+        required: true
+        schema:
+          type: string
+          format: date
+        in: query
+      - name: end_date
+        required: true
+        schema:
+          type: string
+          format: date
+        in: query
+      responses:
+        '200':
+          description: Dates available for each event type.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  qa_checks:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        outcome:
+                          type: string
+                        type_of_query_or_check:
+                          type: string
+                        cdr_date:
+                          type: string
+                          format: date
+        '400':
+          description: Bad request
+        '401':
+          description: Unauthorized.
+        '404':
+          description: No QA checks of type specified found on date
+        '500':
+          description: Server error.
+      summary: Returns QA values for a given check type and call id between two dates
+    """
+    current_user.can_get_qa()
+    return await get_qa_checks(
+        cdr_type, check_id, request.args.get("start_date"), request.args.get("end_date")
+    )
+
+
+@blueprint.route("/<cdr_type>/<check_id>/<check_date>")
+@jwt_required
+async def get_qa_on_date(cdr_type, check_id, check_date):
+    """
+    Returns QA values for a given check type and call id on a date
+    ---
+    get:
+      parameters:
+      - name: cdr_type
+        in: path
+        required: true
+        schema:
+          type: string
+      - name: check_id
+        in: path
+        required: true
+        schema:
+          type: string
+      - name: check_date
+        in: path
+        required: true
+        schema:
+          type: string
+          format: date
+      responses:
+        '200':
+          description: Dates available for each event type.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  qa_checks:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        outcome:
+                          type: string
+                        type_of_query_or_check:
+                          type: string
+                        cdr_date:
+                          type: string
+                          format: date
+        '400':
+          description: Bad request
+        '401':
+          description: Unauthorized.
+        '404':
+          description: No QA checks of type specified found on date
+        '500':
+          description: Server error.
+      summary: Returns QA values for a given cdr and check type on a given date
```

Comment:
```suggestion
      summary: Get QA check result for a given cdr and check type on a given date
```

Suggest changing to Get from Returns because then it is in keeping with the other endpoint descriptions

Reply:
Also `s/result/outcome` for consistency with what's in the response.